src/main/java/com/kryptnostic/kindling/search/ConductorElasticsearchImpl.java

```diff
@@ -686,23 +686,33 @@ public boolean createEntityData( UUID entitySetId, String entityId, Map<UUID, Ob
         String indexName = SECURABLE_OBJECT_INDEX_PREFIX + entitySetId.toString();
         String typeName = SECURABLE_OBJECT_TYPE_PREFIX + entitySetId.toString();
 
-        String scriptStr = "";
+        StringBuilder builder = new StringBuilder();
         Map<String, Object> paramValues = Maps.newHashMap();
-
         for ( Entry<UUID, Object> entry : propertyValues.entrySet() ) {
             List<Object> values = Lists.newArrayList( (Set<Object>) entry.getValue() );
             String id = entry.getKey().toString();
             paramValues.put( id, values );
             for ( int i = 0; i < values.size(); i++ ) {
                 String paramName = id + "_" + String.valueOf( i );
-                scriptStr += "if (!ctx._source['" + id + "'].contains(params['" + paramName + "'])) ctx._source['"
-                        + id
-                        + "'].add(params['" + paramName + "']);";
+                builder.append( "if (ctx._source['" ); 
```

Comment:
...and now it's not?? eclipse you confuse me. well thanks, I like that a lot better.